README.md

```diff
@@ -225,7 +197,7 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
Please remove this line because it's only an instruction of what you can put here